tls/s2n_signature_scheme.h

```diff
@@ -74,3 +75,5 @@ extern const struct s2n_signature_scheme s2n_rsa_pss_rsae_sha512;
 extern const struct s2n_signature_preferences s2n_signature_preferences_20140601;
 extern const struct s2n_signature_preferences s2n_signature_preferences_20200207;
 extern const struct s2n_signature_preferences s2n_signature_preferences_null;
+
+extern const struct s2n_signature_preferences s2n_certificate_signature_preferences_20190801;
```

Comment:
Why backdate? Why not 202011\<today\>?

Reply:
I think there's some confusion about whether or not to backdate preferences to the security policy or not. I don't have to, I just saw other preferences doing that.

Reply:
There was some ambiguity when we combined cipher preferences + signature preferences into security policies, but going forwards we should probably date things correctly to avoid more confusion.